cmake/faceliftMacros.cmake

```diff
@@ -483,7 +483,7 @@ endmacro()
 
 function(facelift_add_library TARGET_NAME)
 
-    _facelift_parse_target_arguments("SYSTEM;STATIC;SHARED;OBJECT;MODULE;MONOLITHIC_SUPPORTED;NO_MONOLITHIC" ""
+    _facelift_parse_target_arguments("SYSTEM;STATIC;SHARED;OBJECT;MODULE;MONOLITHIC_SUPPORTED;NO_MONOLITHIC" "LINK_DEPENDS_NO_SHARED"
```

Comment:
LINK_DEPENDS_NO_SHARED applies to all target types => move into _facelift_parse_target_arguments